source/dhtslib/tabix.d

```diff
@@ -108,12 +116,12 @@ struct TabixIndexedFile {
             // but first row was preloaded in this.next)
             private bool active;
 
-            this(htsFile *fp, tbx_t *tbx, const(char)[] r)
+            this(htsFile *fp, tbx_t *tbx, string chrom, Zbho coords)
```

Comment:
again backwards compat needed

Reply:
So here, if this previously took const(char)[] it means it took a region string, no? In this case, should it not take ChromCoords (or whatever) ?

Reply:
It should, this was an oversight. Will fix.

Reply:
addressed with b55d325